build/prepare-versioning.ps1

```diff
@@ -35,9 +35,9 @@ $nugetVersion = "{0}.{1}.{2}" -f ($versionTokens[0], $versionTokens[1], $version
 
 Write-Host "=========================="
 Write-Host "Versioning .nuspec file..."
-$filename = $env:TO_PACK_TARGET + "\" + $env:LIBRARY_NAME + ".nuspec"
+$filename = "build\Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec"
 $content = Get-Content $filename
-$newContent = $content -replace "REPLACE_DURING_BUILD", $nugetVersion
+$newContent = $content -replace "<version>(.*)</version>", "<version>$nugetVersion</version>"
```

Comment:
The _version_ in the hard-coded _nuspec_ file was modified about. This change is required to the versioning script so it correctly updates the file.